imap_processing/tests/swe/test_swe_l2.py

```diff
@@ -256,8 +256,13 @@ def test_put_data_into_angle_bins():
 
 
 @patch("imap_data_access.processing_input.ProcessingInputCollection.get_file_paths")
+@patch(
+    "imap_processing.spice.spin.get_spacecraft_to_instrument_spin_phase_offset",
+    return_value=153 / 360,
```

Comment:
This is a nice way to approach this for now. I think we might want to open an issue for @tech3371 to update the expected values at some point though...